docs/_includes/ds6/combobox.html

```diff
@@ -0,0 +1,181 @@
+                        <span class="combobox__status"></span>
+                    </div>
+                    <div class="combobox__option" role="option">
+                        <span>Option 3</span>
+                        <span class="combobox__status"></span>
+                    </div>
+                </div>
+                <select name="nativeOptions" hidden>
+                    <option value="item1">Option 1</option>
+                    <option value="item2">Option 2</option>
+                    <option value="item3">Option 3</option>
+                </select>
+            </span>
+        </div>
+        {% highlight html %}
+<span class="combobox">
+    <span class="combobox__control">
+        <input type="text" value="Option 1" readonly role="combobox" aria-haspopup="combobox" aria-owns="combobox1"/>
+        <span class="combobox__arrow" aria-hidden="true"></span>
+    </span>
+    <div id="combobox1" class="combobox__options" role="listbox">
+        <div class="combobox__option" role="option" aria-selected="true">
+            <span>Option 1</span>
+            <span class="combobox__status"></span>
+        </div>
+        <div class="combobox__option" role="option">
+            <span>Option 2</span>
+            <span class="combobox__status"></span>
+        </div>
+        <div class="combobox__option" role="option">
+            <span>Option 3</span>
+            <span class="combobox__status"></span>
+        </div>
+    </div>
+    <select name="nativeOptions" hidden>
+        <option value="item1">Option 1</option>
+        <option value="item2">Option 2</option>
+        <option value="item3">Option 3</option>
+    </select>
+</span>
+        {% endhighlight %}
+    </div>
+
+    <h3>Selected combobox</h3>
+    <div class="demo">
+        <div class="demo__inner" style="max-width: 200px;">
+            <span class="combobox">
+                <span class="combobox__control">
+                    <input type="text" value="Option 2" readonly role="combobox" aria-haspopup="combobox" aria-owns="combobox2"/>
+                    <span class="combobox__arrow" aria-hidden="true"></span>
+                </span>
+                <div id="combobox2" class="combobox__options" role="listbox">
+                    <div class="combobox__option" role="option">
+                        <span>Option 1</span>
+                        <span class="combobox__status"></span>
+                    </div>
+                    <div class="combobox__option" role="option" aria-selected="true">
+                        <span>Option 2</span>
+                        <span class="combobox__status"></span>
+                    </div>
+                    <div class="combobox__option" role="option">
+                        <span>Option 3</span>
+                        <span class="combobox__status"></span>
+                    </div>
+                </div>
+                <select name="nativeOptions" hidden>
```

Comment:
What do you think to us removing the under-the-hood `<select>` from the documentation of skin? Is it helpful or confusing?

Reply:
Oh, so you could, in theory, use a combobox without a select?

Reply:
Yep. For pure client-side use case.

Reply:
👍

Reply:
So, I removed this from the demo code, but left it in the actual example. Thoughts? Should we rip it out altogether?

Reply:
I think it's all or nothing.

Reply:
Okay. Let me look at removing it altogether.